### PR TITLE
Feature/health chk post edges

### PR DESCRIPTION
**Pull request by carlosmondra**
Performs sanity check after Edges have been POSTed in bulk

### CODE REVIEW

**Comment by carlosmondra** [comment]
Before approving, I was thinking that this check should be done in the openreview backend not here. What do you think?

**Comment by melisabok** [comment]
Yes, let's do the checking in the backend too.

File: openreview/conference/matching.py [hunk 1]
```diff
@@ -182,7 +183,12 @@ def _build_conflicts(self, submissions, user_profiles):
                         writers=[self.conference.id],
                         signatures=[self.conference.id]
                     ))
-            openreview.tools.post_bulk_edges(client=self.client, edges=edges)
+        openreview.tools.post_bulk_edges(client=self.client, edges=edges)
+        # Perform sanity check
+        edges_iter = openreview.tools.iterget_edges(self.client, invitation=invitation.id)
+        edges_posted = sum(1 for _ in edges_iter)
```
Comment (melisabok) [inline openreview/conference/matching.py]: we usually convert the iterator to a list:

```suggestion
        edges_posted = len(list(edges_iter))
```

File: openreview/conference/matching.py [hunk 2]
```diff
@@ -223,6 +229,11 @@ def _build_tpms_scores(self, tpms_score_file, submissions, user_profiles):
                     ))
 
         openreview.tools.post_bulk_edges(client=self.client, edges=edges)
+        # Perform sanity check
+        edges_iter = openreview.tools.iterget_edges(self.client, invitation=invitation.id)
+        edges_posted = sum(1 for _ in edges_iter)
```
Comment (melisabok) [inline openreview/conference/matching.py]: same here

File: openreview/conference/matching.py [hunk 3]
```diff
@@ -250,7 +261,12 @@ def _build_scores(self, score_invitation_id, score_file, submissions):
                     signatures=[self.conference.id]
                 ))
 
-        openreview.tools.post_bulk_edges(client=self.conference.client, edges=edges)
+        openreview.tools.post_bulk_edges(client=self.client, edges=edges)
+        # Perform sanity check
+        edges_iter = openreview.tools.iterget_edges(self.client, invitation=invitation.id)
+        edges_posted = sum(1 for _ in edges_iter)
```
Comment (melisabok) [inline openreview/conference/matching.py]: same here, I forgot to suggest the changes.

File: openreview/conference/matching.py [hunk 4]
```diff
@@ -283,6 +299,11 @@ def _build_subject_area_scores(self, submissions):
                     ))
 
         openreview.tools.post_bulk_edges(client=self.conference.client, edges=edges)
+        # Perform sanity check
+        edges_iter = openreview.tools.iterget_edges(self.conference.client, invitation=invitation.id)
+        edges_posted = sum(1 for _ in edges_iter)
```
Comment (melisabok) [inline openreview/conference/matching.py]: same here

**Pull request merged**